skbio/stats/tests/test_composition.py

```diff
@@ -269,6 +355,136 @@ def test_centralize(self):
                             np.array([[2, 2, 6],
                                       [4, 4, 2]]))
 
```

Comment:
Here's the test for [`tau`](https://github.com/biocore/scikit-bio/pull/1114/files#diff-826d59e630d307581ca0d6b8e9181f68R597).  Did you have something else in mind?